scripts.js

```diff
@@ -197,6 +197,7 @@
 
   function helixQuery(appId, key) {
     return async (queries, hitsPerPage) => {
+      if (!queries || queries.length === 0) return {};
```

Comment:
That function usually seems to return an object `{ hits: [...] }` - could a client be confused if it returns an object without `hits` property?

Reply:
Hmm, good question. So far we only try to access the `hits` property in one place, `setupSearch()`, using object destructuring, and we fail gracefully if is undefined.